### DISCUSSION TITLE
M4DModule chart name does not append values.image.tag

### DISCUSSION THREAD

**Discussion by elsalant** [fybrik/fybrik#550] [Help]
The M4DModule asks for both a chart name AND image tag.  However, the chart name itself requires the image tag on it.  This is unexpected; if the image.tag is given, why it this suffix required on the chart name?
spec:
  chart:
    name: ghcr.io/elsalant/hello-world-read-module-chart:v1
    values:
      image.tag: v1

**Comment by roee88** [comment]
A chart image is tagged independently of whatever it contains (which could include a tagged container, 7 tagged containers, or not at all). 

`values` is for overriding parameters from the default `values.yaml` of the chart. In your example the chart values happen to have an `image.tag` value and your M4DModule happens to override it (it doesn't have to).

Usually in Helm the Chart.yaml defines the version of the chart and the version of the deployed application (separate fields) and those are used as the defaults.
